src/logic.js

```diff
@@ -487,8 +489,26 @@ define([
                     load[path] = [prop];
                 }
             });
-
-            return {load: load};
+            var saveToCaseQuestions = _.filter(this.form.getMugList(), function(mug) {
+                return mug.__className === "SaveToCase";
```

Comment:
is this ~what you had in mind? https://github.com/dimagi/Vellum/pull/850/commits/3e29c61b3df6a9e3f82470421dc4fecc38d1a6ad